MANIFEST.in

```diff
@@ -2,6 +2,9 @@ include CHANGES.rst
 include LICENSE
 include ignore_words.txt
 include requirements.txt
+recursive-include sphinx_gallery *.css
+recursive-include sphinx_gallery *.png
+recursive-include sphinx_gallery *.svg
```

Comment:
Again a bit of information loss I think -- the `setup.py` was more specifically scoped to `_static`, no?

Reply:
You are right, I was following the check suggestion

Reply:
```suggestion
include sphinx_gallery/_static/sg_gallery*.css
include sphinx_gallery/ _static/no_image.png
include sphinx_gallery/_static/broken_example.png
include sphinx_gallery/_static/binder_badge_logo.svg
include sphinx_gallery/_static/jupyterlite_badge_logo.svg
```